gen/dcos-config.yaml

```diff
@@ -1347,6 +1347,8 @@ package:
       # Additional Telegraf config for agents
       # Plugin for monitoring mesos container resource consumption
       [[inputs.dcos_containers]]
+        ## The interval at which to collect metrics
```

Comment:
I won't block this PR but it would be great if this could come with a commend describing why 60s was chosen.

This might look something like:

```
# At a 10s interval, we experienced slow downs in the Docker agent.
# The symptom of this was XXX.
# We chose 60s because YYY.
```

I would at least like this PR description or the JIRA to fill in XXX and YYY.

Reply:
@adamtheturtle +1 to this suggestion. cc: @philipnrmn